README.md

```diff
@@ -79,11 +79,64 @@ Here's the whole `config.pbtxt` we use for the `ResizePipeline` example:
         output [
         {
             name: "DALI_OUTPUT_0"
-            data_type: TYPE_FP32
+            data_type: TYPE_UINT8
             dims: [ 224, 224, 3 ]
         }
         ]
 
+You can omit writing most of the configuration file if you specify information about the
+inputs, outputs and max batch size in the pipeline definition.
+Refer to [Configuration auto-complete](#Configuration-auto-complete) for the details about this feature.
+
+## Configuration auto-complete
+
+To simplify the process of deploying a model, Triton Server can infer parts of the
+configuration file from the model file itself. In case of DALI backend, the information
```

Comment:
nitpick for brevity:

```suggestion
To simplify deploying of a model, Triton Server can infer parts of the
configuration file from the model file. In case of DALI backend, the information
```